src/UCTSearch.cpp

```diff
@@ -168,6 +168,7 @@ SearchResult UCTSearch::play_simulation(GameState & currstate,
         if (currstate.get_passes() >= 2) {
             auto score = currstate.final_score();
             result = SearchResult::from_score(score);
+            node->update(result.eval());
```

Comment:
Also needed on line 179

Reply:
that one is done in UCTNode.cpp in create_children

Reply:
That updates `m_net_eval` but `node->update()` accumulates into `m_blackevals`.

Reply:
I think that is correct? The NN output is for side-to-move, and m_net_eval is replaced by 1-m_net_eval when it's white to move to get blackeval.